orchestrator-salt/src/main/resources/salt-common/salt/filecollector/init.sls

```diff
@@ -0,0 +1,88 @@
+{%- from 'telemetry/settings.sls' import telemetry with context %}
+{%- from 'filecollector/settings.sls' import filecollector with context %}
+{%- from 'fluent/settings.sls' import fluent with context %}
+
+install_fluent_logger:
+  cmd.run:
+    - name: pip install fluent-logger>=0.9.6 --ignore-installed
```

Comment:
We need new images with these packages otherwise the no-internet test will fail and it will be a blocker for promotion.

Reply:
these are not in highstate (you need to trigger this tasks by API) + already inside the images (see: https://github.com/hortonworks/cloudbreak-images/commit/67bec331ea94a47626f8297d338aaa0226cd07bb), will be removed later